llm-structured-extraction/2.tips.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
<p>Nice example of streaming here, should we add some markdown to explain it? Haven't seen the video yet, but just reviewing the notebooks took me a while to figure out the difference between this cell and the one above. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Just saw the video and it makes it pretty clear, but I think a short markdown comment above the second cell could help (no need to re-record the video)